packages/create-contentful-app/template/README.md

```diff
@@ -0,0 +1,62 @@
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
```

Comment:
We should customize this README to be more Contentful specific.

Reply:
Left this one for the documentation ticket. I think there's a right balance on mentioning we're using CRA which will give devs freedom on fine tuning their bootstrapped app using CRA documentation.